codegen/templates/main.tmpl

```diff
@@ -42,8 +43,11 @@ func getConfig() *zanzibar.StaticConfig {
 
 func createGateway() (*zanzibar.Gateway, error) {
 	config := getConfig()
-	
-	gateway, _, err := service.CreateGateway(config, nil)
+	opts := &zanzibar.Options{
```

Comment:
Since `app.AppOptions` is already an instance of `*zanzibar.Options`, you don't need to copy the fields over, just pass it as-is to `CreateGateway(config, app.AppOptions)`